internal/security/config/command/proxy/oauth2/command.go

```diff
@@ -53,19 +54,23 @@ func NewCommand(
 	flagSet := flag.NewFlagSet(CommandName, flag.ContinueOnError)
 	flagSet.StringVar(&dummy, "confdir", "", "") // handled by bootstrap; duplicated here to prevent arg parsing errors
 
+	flagSet.StringVar(&cmd.jwt, "jwt", "", "The JWT for use when accessing the Kong Admin API")
 	flagSet.StringVar(&cmd.clientID, "client_id", "", "Optional manually-specified OAuth2 client_id. Will be generated if not present. Equivalent to a username.")
 	flagSet.StringVar(&cmd.clientSecret, "client_secret", "", "Optional manually-specified OAuth2 client_secret. Will be generated if not present. Equivalent to a password.")
 
 	err := flagSet.Parse(args)
 	if err != nil {
-		return nil, fmt.Errorf("Unable to parse command: %s: %w", strings.Join(args, " "), err)
+		return nil, fmt.Errorf("unable to parse command: %s: %w", strings.Join(args, " "), err)
 	}
 	if cmd.clientID == "" {
 		return nil, fmt.Errorf("%s proxy oauth2: argument --client_id is required", os.Args[0])
 	}
 	if cmd.clientSecret == "" {
 		return nil, fmt.Errorf("%s proxy oauth2: argument --client_secret is required", os.Args[0])
 	}
+	if cmd.jwt == "" {
+		return nil, fmt.Errorf("%s proxy oauth2: argument --admin_api_jwt is required", os.Args[0])
```

Comment:
nit pick: the error message is still saying --admin_api_jwt but actually is --jwt

Reply:
Good find...I'll adjust it.

Reply:
Pushed up.